### PR TITLE
Updates to create good looking PDF files from URLs

### PR DESCRIPTION
**Pull request by kylebjones**
Modified some settings to make the PDF print correctly.

### CODE REVIEW

File: CertiPay.PDF/IPDFService.cs [hunk 1]
```diff
@@ -184,7 +180,7 @@ public Settings()
                 this.Timeout = TimeSpan.FromSeconds(60);
                 this.RetryCount = 1;
                 this.ContentCount = 36;
-                this.UseScript = true;
+                this.UseScript = false;
```
Comment (mattgwagner) [inline CertiPay.PDF/IPDFService.cs]: Why'd we change the default on this?

File: CertiPay.PDF/IPDFService.cs [hunk 2]
```diff
@@ -82,7 +78,7 @@ public byte[] CreatePdf(PDFService.Settings settings)
                     }
 
                     // Render the web page by uri and return the image id for chaining
-                    imageId = pdf.AddImageUrl(uri, paged: true, width: 0, disableCache: false);
+                    imageId = pdf.AddImageUrl(uri, true, intHTMLWidth, true);
```
Comment (mattgwagner) [inline CertiPay.PDF/IPDFService.cs]: Can you re-add the named parameters so it's more readable without knowing the API?

File: CertiPay.PDF/IPDFService.cs (hunk 1 above)
Comment (kylebjones) [inline CertiPay.PDF/IPDFService.cs]: ABCPDF recommends you dont have scripts set to true for security

Comment (mattgwagner) [inline CertiPay.PDF/IPDFService.cs]: Good reason; when we get back in there, we'll throw a parameter into the PDF service to re-enable.

File: CertiPay.PDF/IPDFService.cs (hunk 2 above)
Comment (kylebjones) [inline CertiPay.PDF/IPDFService.cs]: Added back

**Pull request merged**